CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bugs fixed
 
 * [#520](https://github.com/clojure-emacs/clojure-mode/issues/508): Fix allow `clojure-align-cond-forms` to recognize qualified forms.
+* Enhance add arity refactoring to support reader conditionals, letfn, fn, defmacro, defmethod, defprotocol, reify and proxy in addition to defn.
```

Comment:
I'd split this to an entry about the bugfix and a different one about adding support for more forms.